Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -158,25 +158,25 @@ public static RouteGroupBuilder UseOrganisationLookupEndpoints(this RouteGroupBu
                 return operation;
             });
         app.MapGet("/lookup",
-                async ([FromQuery] string name, IUseCase<string, Model.Organisation?> useCase) =>
-                await useCase.Execute(name)
-                    .AndThen(organisation => organisation != null ? Results.Ok(organisation) : Results.NotFound()))
-            .Produces<Model.Organisation>(StatusCodes.Status200OK, "application/json")
-            .Produces<ProblemDetails>(StatusCodes.Status401Unauthorized)
-            .Produces<ProblemDetails>(StatusCodes.Status404NotFound)
-            .Produces<ProblemDetails>(StatusCodes.Status500InternalServerError)
-            .WithOpenApi(operation =>
-            {
-                operation.OperationId = "LookupOrganisation";
-                operation.Description = "Find an organisation.";
-                operation.Summary = "Find an organisation.";
-                operation.Tags = new List<OpenApiTag> { new() { Name = "Organisation - Lookup" } };
-                operation.Responses["200"].Description = "Organisations Associated.";
-                operation.Responses["401"].Description = "Valid authentication credentials are missing in the request.";
-                operation.Responses["404"].Description = "Organisation not found.";
-                operation.Responses["500"].Description = "Internal server error.";
-                return operation;
-            });
+           async ([FromQuery] string query, IUseCase<string, Model.Organisation?> useCase) =>
```

Comment:
We were suppose to add a new query parameter to support both `name` and `identifier`.
It only works here as we can distinguish between the name and identifier, but what if we have to support another query type in a week?

It's actually specified in the ticket to support this kind of query in addition to name lookup: `/organisation/lookup?identifier=GB-COH:9543428757`

Reply:
I will update it now

Reply:
All updated now.